omnigibson/utils/python_utils.py

```diff
@@ -678,7 +678,6 @@ def deserialize(cls, state):
         """
         # Sanity check the idx with the expected state size
         state_dict, idx = cls._deserialize(state=state)
-        assert cls.state_size is not None, "State size must be specified by subclass!"
```

Comment:
why is this assert needed? doesn't this core implementation raise a `NotImplementedError` by default?

Reply:
Right. I am not sure why this is needed. That's why I removed it.